.github/workflows/tests.yml

```diff
@@ -1,6 +1,12 @@
 name: tests
 
-on: push
+on:
```

Comment:
I think is better to run tests on every branch. We may have checks that prevent merges to master when tests fail but I think is better to know if they are failing in any other PR.

Reply:
I’m not sure I follow. This PR setup runs tests/checks on PRs targeting `main` and direct pushes to it. Are you suggesting we should also run tests on:

	1.	PRs targeting any branch?
	2.	Any commit pushed to any branch?

If that’s the case, there could be some downsides to consider:

	•	Duplicate test runs
	•	Increased resource usage
	•	More notifications (added noise)

Reply:
Yeah, the problem is when developing an epic (the PRs would target a branch different from `dev` or `master`). I would let the rule to:
1. Target any push in `dev` or `main`
2. Target any PR no matter the branch.

Reply:
Done!